Makefile

```diff
@@ -19,7 +19,7 @@ build-dev: build
 	mv ${BINARY} ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${PKG_NAME}/${VERSION}/${OS_ARCH}
 
 testacc: fmtcheck
-	TF_ACC=1 go test $(TEST) -v -count 3 $(TESTARGS) -timeout 120m
+	TF_ACC=1 go test $(TEST) -v -count 1 $(TESTARGS) -timeout 120m
```

Comment:
Hey @tuteng I noticed the tests almost always failing with -count 3 set, I adjusted this to 1 and the e2e tests are much more consistent. Do we know why this was set as 3 to begin with?

Reply:
@tuteng just wanna bump incase you missed this